docs/code_ref/time_lag.rst

```diff
@@ -0,0 +1,8 @@
+********
+time_lag
```

Comment:
Controversial but what if we just called this lag?

Reply:
I think that would be fine. The  method is widely known in the community as the "time lag method", but as long as the function is still named that I think it's fine.

Reply:
For now, I'd prefer to just call this `time_lag`, but happy to reconsider renaming it in the future.